Fix Synology entity ID naming (#17238)
home-assistant/home-assistant.github.io: source/_integrations/synology_dsm.markdown

## Sensors

### CPU Utilisation sensors

> Entities reporting the current and combined CPU utilisation of the NAS. There are sensors the report the current CPU load, separated by User, System and other. By default only the User sensor is enabled.
> 
> There are also combined CPU load sensors. These report the total CPU load for the entire NAS. Available as current, 1min, 5min and 15min load sensors. By default the 1min load sensor is disabled.

Reviewer: ```suggestion
Entities reporting the current and combined CPU utilization of the NAS. There are sensors the report the current CPU load, separated by User, System and others. By default, only the User sensor is enabled.

There are also combined CPU load sensors. These report the total CPU load for the entire NAS. Available as current, 1min, 5min and 15min load sensors. By default the 1min load sensor is disabled.
```

### Memory Utilisation sensors

> Entities reporting the current and combined memory and swap utilisation of the NAS. These sensors include the total installed amount, the currently free amount and the % of memory used. 

Reviewer: ```suggestion
Entities reporting the current and combined memory and swap utilization of the NAS. These sensors include the total installed amount, the currently free amount and the % of memory used. 
```

### Network sensors

Entities reporting the current network transfer rates of the NAS. Both upload and download sensors are available.

### General sensors

Entities reporting the internal temperature and the uptime of the NAS. The uptime sensor is disabled by default.

### Disk sensors

Entities reporting the internal temperature, status (as shown in Synology DSM) and SMART status for each drive inside the NAS. The SMART status sensor is disabled by default.

### Volume sensors

Entities reporting status, total size (TB), used size (TB), % of volume used, average disk temperature and maximum disk temperature for each volume inside the NAS. By default the total size and maximum disk temperature sensors are disabled.

## Binary sensors

### General sensors

Entities reporting the update and security status of the NAS.

### Disk sensors

> Similar to the [normal disk sensors](#disk-sensors), there are binary sensors reporting each drive's status. These sensors report if a drive has exceeded the maximum threshold for detected bad sectors and if a drive has dropped below the threshold for it's remaining life.

Reviewer: ```suggestion
Similar to the [normal disk sensors](#disk-sensors), there are binary sensors reporting each drive's status. These sensors report if a drive has exceeded the maximum threshold for detected bad sectors and if a drive has dropped below the threshold for its remaining life.
```

## Sensors

Utilisation:
- `cpu_load_other`: Displays unspecified (that is, not user or system) load in percentage.
- `cpu_load_user`: Displays user load in percentage.
- `cpu_load_system`: Displays system load in percentage.
- `cpu_load_total`: Displays combined load in percentage.
- `cpu_load_1min`: Displays maximum load in past minute.
- `cpu_load_5min`: Displays maximum load in past 5 minutes.
- `cpu_load_15min`: Displays maximum load in past 15 minutes.
- `memory_usage_real`: Displays percentage of memory used.
- `memory_size`: Displays total size of memory in MB.
- `memory_cached`: Displays total size of cache in MB.

> - `memory_available_swap`: Displays total size of available swap in MB.

Reviewer: This integration uses a configuration flow and the entity registry. The entity names are therefore never guaranteed. It is better to describe a list of provided sensors instead.
